packages/eslint-config/nextjs.js

```diff
@@ -25,6 +25,11 @@ module.exports = {
                         importNames: ["default"],
                         message: "Please use Image from @comet/cms-site instead",
                     },
+                    {
+                        name: "Alert",
+                        importNames: ["default"],
+                        message: "Please use Alert from @comet/admin/lib/alert/Alert instead",
+                    },
```

Comment:
@johnnyomair Is it correct like that?

Reply:
I guess it should be

```suggestion
                    {
                        name: "@mui/material",
                        importNames: ["Alert"],
                        message: "Please use Alert from @comet/admin instead",
                    },
```

you can derive it from the import `import { Alert } from "@mui/material";`